### PR TITLE
server: readable fields

### PR DESCRIPTION
**Pull request by vbatts**
`git grep -w images` or `git grep -w storage` needs to be more useful.

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -29,20 +29,20 @@ const (
...
-	podNameIndex *registrar.Registrar
-	podIDIndex   *truncindex.TruncIndex
-	ctrNameIndex *registrar.Registrar
-	ctrIDIndex   *truncindex.TruncIndex
-	imageContext *types.SystemContext
+	config        Config
+	runtime       *oci.Runtime
+	store         sstorage.Store
+	imageServer   storage.ImageServer
+	runtimeServer storage.RuntimeServer
```
Comment (mrunalp) [inline server/server.go]: We also have a runtime field. Should we call these storageRuntimeService and storageImageService?

Comment (vbatts) [inline server/server.go]: I can do that. I'll take a look tomorrow.

**Comment by mrunalp** [comment]
Thanks

**Comment by vbatts** [comment]
updated. PTAL

**Comment by mrunalp** [comment]
LGTM

**Comment by mrunalp** [comment]
@sameo @runcom PTAL

**Comment by runcom** [comment]
LGTM

**Pull request merged**